ros_ign_bridge/include/ros_ign_bridge/convert/ros_ign_interfaces.hpp

```diff
@@ -19,13 +19,15 @@
 #include <ros_ign_interfaces/msg/joint_wrench.hpp>
 #include <ros_ign_interfaces/msg/contact.hpp>
 #include <ros_ign_interfaces/msg/contacts.hpp>
+#include <ros_ign_interfaces/msg/dataframe.hpp>
 #include <ros_ign_interfaces/msg/light.hpp>
 
 // Ignition messages
 #include <ignition/msgs/entity.pb.h>
 #include <ignition/msgs/joint_wrench.pb.h>
 #include <ignition/msgs/contact.pb.h>
 #include <ignition/msgs/contacts.pb.h>
+#include <ignition/msgs/dataframe.pb.h>
```

Comment:
The `galactic` branch needs to compile against Edifice and Fortress. So we'll need `ifdef`s here since this message isn't in Edifice.

Reply:
I added a `HAVE_DATAFRAME` variable that is only set when ignition-msgs 8.4.0 or greater is installed in 565a6bb

This should allow edifice and fortress to build so that we can go ahead and merge this to galactic.  Once ign-msgs > 8.4.0 gets synced out, this functionality will come along without additional modifications.